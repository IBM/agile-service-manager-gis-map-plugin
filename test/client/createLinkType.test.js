// Import the module under test
import {createLinkType} from '../../src/scripts/client/createLinkType';

describe('createLinkType', function() {
    test('create', function() {        
        expect(createLinkType('myType')).toMatchSnapshot();
    });
});