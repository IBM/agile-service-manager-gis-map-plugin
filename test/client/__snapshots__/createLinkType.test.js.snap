// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createLinkType create 1`] = `
Object {
  "clusterGroup": NewClass {
    "_initHooksCalled": true,
    "_layers": Object {},
    "options": Object {},
  },
  "linkType": "myType",
  "links": Object {},
}
`;
